app/services/induction/change_induction_record.rb

```diff
@@ -12,7 +12,7 @@ def call
           start_date: time_now,
           school_transfer: false,
         }
-        new_record = induction_record.dup
+        new_record = InductionRecord.new(induction_record.attributes.except("id", "end_date"))
```

Comment:
The intention was to copy _everything_ and apply the changes you supply to the call.  If you want to set/enforce `end_date` to `nil` then you can pass it in as part of the changes. I think this will break some assumptions made elsewhere if we don't do this.